[{"tags": ["python-3.x", "pytorch"], "owner": {"account_id": 16927087, "reputation": 85, "user_id": 12242415, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/7cb3c40ebe9fbbcb6bac821d31af78b6?s=256&d=identicon&r=PG&f=1", "display_name": "MarayJay", "link": "https://stackoverflow.com/users/12242415/marayjay"}, "is_answered": true, "view_count": 255, "answer_count": 1, "score": 1, "last_activity_date": 1610459056, "creation_date": 1610449925, "last_edit_date": 1610451460, "question_id": 65682794, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/65682794/pytorch-batchnorm3d-instancenorm3d-not-working-when-data-size-1-c-1-1-1", "title": "Pytorch BatchNorm3d / InstanceNorm3d not working when data size (1,C,1,1,1)"}, {"tags": ["chainer"], "owner": {"account_id": 11515626, "reputation": 263, "user_id": 8439076, "user_type": "registered", "accept_rate": 63, "profile_image": "https://www.gravatar.com/avatar/2a76d4b910948e3655ef956abc062054?s=256&d=identicon&r=PG&f=1", "display_name": "machen", "link": "https://stackoverflow.com/users/8439076/machen"}, "is_answered": false, "view_count": 67, "answer_count": 1, "score": 0, "last_activity_date": 1526535525, "creation_date": 1526109829, "question_id": 50303841, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/50303841/how-to-use-batchnormalization3d-in-chainer", "title": "How to use BatchNormalization3D in chainer?"}, {"tags": ["python", "neural-network", "deep-learning", "pytorch", "batch-normalization"], "owner": {"account_id": 4958744, "reputation": 16295, "user_id": 3990607, "user_type": "registered", "accept_rate": 77, "profile_image": "https://i.stack.imgur.com/0ZRtB.png?s=256&g=1", "display_name": "patapouf_ai", "link": "https://stackoverflow.com/users/3990607/patapouf-ai"}, "is_answered": true, "view_count": 21962, "accepted_answer_id": 47202205, "answer_count": 2, "score": 12, "last_activity_date": 1580675809, "creation_date": 1510218830, "last_edit_date": 1520944476, "question_id": 47197885, "content_license": "CC BY-SA 3.0", "link": "https://stackoverflow.com/questions/47197885/how-to-do-fully-connected-batch-norm-in-pytorch", "title": "How to do fully connected batch norm in PyTorch?"}, {"tags": ["tensorflow", "keras", "pytorch"], "owner": {"account_id": 15573539, "reputation": 1638, "user_id": 11235205, "user_type": "registered", "profile_image": "https://i.stack.imgur.com/cTAqb.jpg?s=256&g=1", "display_name": "Minh-Long Luu", "link": "https://stackoverflow.com/users/11235205/minh-long-luu"}, "is_answered": false, "view_count": 1231, "answer_count": 1, "score": 0, "last_activity_date": 1604221878, "creation_date": 1604215100, "question_id": 64629522, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/64629522/pytorch-batchnorm1d-2d-3d-and-tensorflow-keras-batchnormalization", "title": "PyTorch BatchNorm1D, 2D, 3D and TensorFlow/Keras BatchNormalization"}, {"tags": ["python", "deep-learning", "pytorch"], "owner": {"account_id": 4577878, "reputation": 532, "user_id": 3715862, "user_type": "registered", "accept_rate": 9, "profile_image": "https://i.stack.imgur.com/rp7dF.jpg?s=256&g=1", "display_name": "Echo", "link": "https://stackoverflow.com/users/3715862/echo"}, "is_answered": false, "view_count": 318, "answer_count": 1, "score": 1, "last_activity_date": 1639946518, "creation_date": 1639944746, "last_edit_date": 1639946518, "question_id": 70414967, "content_license": "CC BY-SA 4.0", "link": "https://stackoverflow.com/questions/70414967/valueerror-target-size-torch-size2-must-be-the-same-as-input-size-torch", "title": "ValueError: Target size (torch.Size([2])) must be the same as input size (torch.Size([504, 2]))"}, {"tags": ["tensorflow"], "owner": {"account_id": 11406834, "reputation": 31, "user_id": 8362050, "user_type": "registered", "profile_image": "https://www.gravatar.com/avatar/bdba87fdc838f492f65a7e302de0907d?s=256&d=identicon&r=PG&f=1", "display_name": "Tramac", "link": "https://stackoverflow.com/users/8362050/tramac"}, "is_answered": true, "view_count": 1198, "answer_count": 1, "score": 3, "last_activity_date": 1500973975, "creation_date": 1500969417, "last_edit_date": 1500973302, "question_id": 45297227, "content_license": "CC BY-SA 3.0", "link": "https://stackoverflow.com/questions/45297227/valueerror-no-gradients-provided-for-any-variable-in-tensorflow", "title": "ValueError: No gradients provided for any variable in Tensorflow"}]